Items/Consumable/LifeStone.cs

```diff
@@ -35,16 +30,11 @@ public override void SetDefaults()
 			Item.useTime = 30;
 			Item.useAnimation = 30;
 			Item.UseSound = SoundID.Item29;
-			Item.consumable = true;
+			Item.consumable = false;
```

Comment:
It does with this change

Short Summary:
• Adds item.Consummable as a check in Quick Heal and Quick Mana
• Is a simpler solution then overriding ConsumeItem(


Porting Notes:
• For Quick heal, Quick Mana non-consummables, it is recommended to use Item.consummable field instead of overriding ConsumeItem()
• Existing overrides will continue to work